ansible/roles/dashmate/tasks/reset_platform.yml

```diff
@@ -1,7 +1,16 @@
 ---
 
+- name: Stop node
```

Comment:
You don't need to stop if you pass --force to reset, isn't it?

Reply:
There is no --force for reset

Reply:
Sorry, actually there is. I will test now

Reply:
fatal: [hp-masternode-1]: FAILED! => changed=false 
  cmd:
  - yarn
  - dashmate
  - reset
  - --platform-only
  - --force
  delta: '0:00:15.096292'
  end: '2023-04-11 04:05:29.797558'
  msg: non-zero return code
  rc: 1
  start: '2023-04-11 04:05:14.701266'
  stderr: |-
    [FAILED] (HTTP code 409) conflict - remove dash_masternode_devnet-samara_drive_abci_data: volume is in use - [b0b8c338238e80b7160a109374fc5d406919e85f4e65636b953443d432ea717a]
    [FAILED] (HTTP code 409) conflict - remove dash_masternode_devnet-samara_drive_abci_data: volume is in use - [b0b8c338238e80b7160a109374fc5d406919e85f4e65636b953443d432ea717a]
  stderr_lines: <omitted>
  stdout: |-
    [STARTED] Reset devnet-samara node
    [STARTED] Check services are not running
    [SKIPPED] Check services are not running
    [STARTED] Remove platform services and associated data
  stdout_lines: <omitted>

TASK [kill_platform : Remove platform docker containers and volumes] ****************************************************************************************************************
changed: [seed-1]

TASK [kill_platform : Delete platform dir] ******************************************************************************************************************************************
changed: [seed-1]

TASK [kill_platform : Remove any remaining docker data] *****************************************************************************************************************************